crates/pop-cli/src/commands/build/parachain.rs

```diff
@@ -1,27 +1,20 @@
 // SPDX-License-Identifier: GPL-3.0
 
 use crate::{cli, style::style};
-use clap::Args;
 use pop_common::Profile;
 use pop_parachains::build_parachain;
 use std::path::PathBuf;
 #[cfg(not(test))]
 use std::{thread::sleep, time::Duration};
 
-#[derive(Args)]
+/// Command to build a parachain with configurable path, package, and profile.
 pub struct BuildParachainCommand {
 	/// Directory path for your project [default: current directory].
```

Comment:
Default part is also no longer necessary as it was there to explain what the command would default to (same for BuildContractsCmd)

Reply:
Good point, feedback, it doesn't make sense to keep the comment as it was before, to display to the user. Refactored in https://github.com/r0gue-io/pop-cli/pull/394/commits/f1963e7bd59df8820b984bc5ae0aaaa95adaaee4